hca/util/__init__.py

```diff
@@ -196,9 +195,11 @@ class SwaggerClient(object):
     #
     timeout_policy = timeout.Timeout(connect=20, read=40)
 
-    def __init__(self, config=None, **session_kwargs):
+    def __init__(self, config=None, swagger_url=None, **session_kwargs):
```

Comment:
What's the advantage of having this parameter point at the `swagger.json` instead of just the base url (the on ending in `/v1`)?

Reply:
In order to be consistent with the tweak config which uses `swagger_url`. Also, I think this makes more sense since:
- the true API host is defined in the swagger definition served at `swagger_url`
- `swagger_url` is used to generate the filename of the cached swagger definition. I think that exposing `swagger_url` makes more sense for developers.

Reply:
Makes sense.